roles/container-engine/containerd/defaults/main.yml

```diff
@@ -70,3 +70,9 @@ containerd_fedora_repo_component: "stable"
 
 # Extra config to be put in {{ containerd_cfg_dir }}/config.toml literally
 containerd_extra_args: ''
+
+# Configure registry auth (if applicable to secure/insecure registries)
+registry_auth: []
```

Comment:
I feel `registry_auth` is too common name if that is only for containerd.
How about `containerd_registry_auth` instead?

Reply:
Sure, done.